Create migrations

f1r3starter opened the pull request:
Update dependencies, fix deprecations notes, add migrations

mbadolato commented:
Just FYI, when running against postgres:

<img width="1020" alt="Screen Shot 2019-04-25 at 9 45 23 PM" src="https://user-images.githubusercontent.com/164615/56784035-7e6e5180-67a3-11e9-9a6b-78da8463e5da.png">

prolic commented:
Thanks for the hint @mbadolato. @f1r3starter can you update the PR?

f1r3starter commented:
@mbadolato  Thanks for pointing out! Fixed
@prolic PR updated, also added commented postgresql block to docker-compose.yml and .env connection, just in case if someone would like to test it with postgres instead of mysql

prolic commented on docker-compose.yml:
```diff
@@ -13,6 +13,18 @@ services:
     volumes:
       - ./vendor/prooph/pdo-event-store/scripts/mysql:/docker-entrypoint-initdb.d
 
+#  postgres:
```
why comment out?

f1r3starter replied:
Because MySql container is used by default

prolic commented on config/migrations/Version20190425071843.php:
```diff
@@ -0,0 +1,75 @@
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+final class Version20190425071843 extends AbstractMigration implements ContainerAwareInterface
```
I think better rename to something like "SetupEventStoreTables"

f1r3starter replied:
[Doctrine documentation recommends](https://www.doctrine-project.org/projects/doctrine-migrations/en/2.0/reference/version-numbers.html) keeping migrations classes naming in that format, to maintain versioning of migrations more easily.

prolic replied:
I'm not sure this is applies here. As this is an event sourced application, there are no migrations at all. You would simply destroy your read model and create a new one.
Migrations here makes only sense as a way to setup the event store. Not to add 'future' migrations.

f1r3starter replied:
From that point of view, I agree with you, renamed

prolic commented:
@prooph/symfony-team please check

mbadolato commented:
Does a migration run when named something arbitrary? I always thought that Doctrine Migrations needed the naming scheme to match up what the latest is in the database and determine what is next and hasn't yet run.

mbadolato commented:
Hmm, yep as suspected. When I named the migration file the same as the last commit, the migration failed. Naming it back to the original and the migration ran.

Or is there a config setting that needs to be changed to allow?

<img width="1136" alt="Screen Shot 2019-05-01 at 9 27 34 PM" src="https://user-images.githubusercontent.com/164615/57056532-fd5a0300-6c57-11e9-9364-69b2e40b4efa.png">
